basis/english/english.factor

```diff
@@ -105,6 +107,48 @@ CONSTANT: plural-to-singular $[ singular-to-plural assoc-invert ]
 
 PRIVATE>
 
+: singular? ( word -- ? )
```

Comment:
It's maybe slower, but maybe this is a better less duplicative definition:

```factor
: singular? ( word -- ? )
    [ singularize ] [ = ] bi ;
```

Reply:
true! I'll change it, but note that I can't use the equivalent `[ pluralize ] [ = ] bi` for `plural?` because `"birds" pluralize` and `"friends" pluralize` give you `birdses` and `friendses` -- not a bug,  just a limitation of trying to generalize English grammar.